### ISSUE TITLE
Movable range

### ISSUE THREAD

**Issue by dxcore35** [sindresorhus/Gifski#209]
After setting the start and stop, when I drag the range I should be able to drag it anywhare - preserving the range duration.
It is usefull when slicing many same duration gifts from same video.

**Comment by sindresorhus** [comment]
I agree that would be useful, but it's not feasible with the current component we use. The trimming component is provided by macOS and we have no control over how it works. We could request Apple to enhance it, but they haven't replied to any of my other enhancement requests about it. So the only other way is to make our own trimming component, which is a lot of effort. But I'll leave this issue open in case anyone would like to work on that.
